test/tools/Modules/HelpersCommon/HelpersCommon.psm1

```diff
@@ -551,7 +551,7 @@ function Get-HelpNetworkTestCases
 
     # Command discovery does not follow symlinks to network locations for module qualified paths
     $networkBlockedError = "CommandNameNotAllowed,Microsoft.PowerShell.Commands.GetHelpCommand"
-    $scriptBlockedError = "ScriptsNotAllowed"
+    $scriptBlockedError = "CommandNotFoundException"
```

Comment:
@TravisEz13 This is a change from your PR https://github.com/PowerShell/PowerShell/pull/20593 which added these tests. As this PR changed default pipeline logic to the V3 branch it now sends the `Get-Help -Name ...` command used by this test as a Command rather than a Script. I don't fully understand the logic done by this test but this gets it working again. I would appreciate it if you could have a look over to ensure this isn't going to be a problem.